parts/k8s/windowsinstallopensshfunc.ps1

```diff
@@ -0,0 +1,45 @@
+function
+Install-OpenSSH {
+    Param(
+        [Parameter(Mandatory = $true)][string] 
+        $SSHKey
+    )
+
+    $adminpath = "c:\ProgramData\ssh\"
+    $adminfile = "administrators_authorized_keys"

```

Comment:
It looks like the `sshd_config` file that is generated with the install of openssh has changed.  Previously there was an entry that looked like:

 ```
AuthorizedKeysFile __PROGRAMDATA__/ssh/administrators_authorized_keys
```

Now it is `AuthorizedKeysFile	.ssh/authorized_keys`. 

For this to work with out a password will  need to change this `adminpath` to `c:\Users\adminuser\.ssh` and `$adminfile` to `authorized_keys` and `adminuser` will need to be the user specified in the `model.json`

Reply:
We may want to make sure we check the content of the `sshd_config` file so we don't break if the generated file changes again?

Reply:
If configured properly should be able to run a command like and not be prompted for a password but be dropped directly into the windows command prompt

```
 ssh -t -i ~/.ssh/id_rsa -o "ProxyCommand ssh -W %h:%p adminuser@<controlnodeip>" adminuser@1003k8s001     
```

where `1003k8s001` is the name of a windows node from `kubectl get nodes` and `controlip` is the public ip address for the control node

Reply:
@jsturtevant maybe we can override the AuthorizedKeysFile enrty in the config everytime?

Reply:
Looks like I was wrong with the `.ssh/authorized_keys`  because I had the key in both places and it was the `administrators_authorized_keys` value that is the correct place.  The script itself has a bug where it silently fails to create that file and permission properly which is what made me think the `administrators_authorized_keys` wasn't working.